bitnami/mariadb-galera/templates/statefulset.yaml

```diff
@@ -98,8 +98,16 @@ spec:
               value: {{ ternary "true" "false" (or .Values.image.debug .Values.diagnosticMode.enabled) | quote }}
             - name: MARIADB_GALERA_CLUSTER_NAME
               value: {{ .Values.galera.name | quote }}
+
+            {{- $fullName := include "common.names.fullname" . }}
+            {{- $releaseName := .Release.Namespace }}
+            {{- $clusterDomain := .Values.clusterDomain }}
+            {{- $initialCluster := list }}
+            {{- range $e, $i := until (int .Values.replicaCount) }}
+            {{- $initialCluster = append $initialCluster (printf "%s-%d.%s-headless.%s.svc.%s" $fullName $i $fullName $releaseName $clusterDomain) }}
+            {{- end }}
             - name: MARIADB_GALERA_CLUSTER_ADDRESS
-              value: "gcomm://{{ template "common.names.fullname" . }}-headless.{{ .Release.Namespace }}.svc.{{ .Values.clusterDomain }}"
+              value: {{ printf "gcomm://%s" (join "," $initialCluster) | quote }}
```

Comment:
will this fix ipv6 as well maybe?

Reply:
Maybe, would you like to test it and provide feedback once this is merged ?